code/controllers/subsystem/garbage.dm

```diff
@@ -95,7 +95,7 @@ SUBSYSTEM_DEF(garbage)
 		dellog += "\tqdel() Count: [I.qdels]"
 		dellog += "\tDestroy() Cost: [I.destroy_time]ms"
 		if (I.hard_deletes)
-			dellog += "\tTotal Hard Deletes [I.hard_deletes]"
+			dellog += "\tTotal Hard Deletes: [I.hard_deletes]"
```

Comment:
I stared at the image you posted in discord for *so* long and never noticed.

Reply:
![image](https://user-images.githubusercontent.com/58055496/116015328-a9d3e300-a5ed-11eb-9954-e2ecfb634c93.png)